src/main.cpp

```diff
@@ -170,7 +170,18 @@ int main(const int argc, const char *argv[])
       // Simplification reduces the number of points used to represent the
       // GeoDivs in the inset, thereby reducing output file sizes and
       // run-times
+
+      std::cerr << "Simplying the input" << std::endl;
+      std::cerr << "Initial number of points = " << inset_state.n_points() << std::endl;
+      std::string output_filename = inset_state.inset_name();
```

Comment:
I was wondering if we need these three extra lines. This information is already printed when the inset_state.simplify() is called.